tests/test_hypersphere.py

```diff
@@ -3,12 +3,16 @@
 """
 
 import unittest
+import scipy.special
```

Comment:
import order: since s < u scipy should come first.

https://www.python.org/dev/peps/pep-0008/#imports